packages/smooth_app/lib/pages/product/hideable_container.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/generic_lib/duration_constants.dart';
+
+class HideableContainer extends StatefulWidget {
```

Comment:
A comment would be helpful

Reply:
> A comment would be helpful

Definitely. The problem is that I have no idea what it's supposed to do and why we need it. I've just refactored blind-folded.